tests/gocase/unit/quit/quit_test.go

```diff
@@ -45,7 +45,7 @@ func TestPipeQuit(t *testing.T) {
 
 		require.NoError(t, c.WriteArgs("PING"))
 		r, err = c.ReadLine()
-		require.EqualError(t, err, "EOF")
+		require.Error(t, err)
```

Comment:
After `QUIT`, the connection can be disposed. So we can possibly receive "read tcp 127.0.0.1:53224->127.0.0.1:36579: read: connection reset by peer" instead of "EOF".